app/controllers/reviews_controller.rb

```diff
@@ -18,6 +20,42 @@ def create
     end
   end
 
+  def upvote
+    @review = Review.find(params[:id])
+    @review.upvote_by current_user
+
+    respond_to do |format|
+      if @review.upvote_by current_user
+        format.json { render json: {
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.